dependency.lic

```diff
@@ -1022,8 +1025,18 @@ class MoonWatch
     @queue << [moon, data]
   end
 
+  def snapshot_seconds_until_load
+    return 60 unless UserVars.moons.first['timer'] && UserVars.sun['timer']
```

Comment:
```
>                                                                   
--- Lich: error: no implicit conversion of String into Integer      
        dependency:1029:in `[]'                                     
        dependency:1029:in `snapshot_seconds_until_load'            
--- Lich: dependency has exited.                                    
```

Reply:
```
>;e echo UserVars.moons.first['timer']                         
--- Lich: exec1 active.                                        
--- Exception: no implicit conversion of String into Integer   
exec1:1:in `[]'                                                
--- Lich: exec1 has exited.                                    
>;e echo UserVars.moons.first                                  
--- Lich: exec1 active.                                        
[exec1: ["katamba", {}]]                                       
--- Lich: exec1 has exited.                                    
>;e echo UserVars.moons.first[1]['timer']                      
--- Lich: exec1 active.                                        
[exec1: ]                                                      
--- Lich: exec1 has exited.                                    
           
```
Is this what we want?

Reply:
Testing this
```
return 60 unless UserVars.moons.reject { |m| m == "visible" }.any? { |_,v| v.key? 'visible' } && UserVars.sun['timer']
```

Reply:
I pushed an update, but we can reject visible, I forgot about that guy.